src/main/java/com/ibm/cloud/cloudant/internal/ServiceFactory.java

```diff
@@ -0,0 +1,25 @@
+package com.ibm.cloud.cloudant.internal;
```

Comment:
This has to be in this package to access protected method `DelegatingAuthenticatorFactory.getAuthenticator`